frontend/src/css/app.sass

```diff
@@ -1,4 +1,7 @@
-// app global css in Sass form
```

Comment:
*(Arbitrary comment location)*

Thanks for tackling this @alexkeating. It's nice to offload a lot of this to a well maintained third party tool, and it all seems to work.

Here are some things I noticed, though. I wonder which can actually be addressed, and if any of them will be showstoppers. Might want input from the rest of the team on the tradeoffs here.

* Ideally we would be able to hide Goerli from the list, but the dapp would still work if the user was connected to Goerli. Is this possible?
* ENS names only resolve when connected to mainnet (@mds1 mentioned this elsewhere too).
* There's not a lot of space to display the ENS name, and even relatively short ones get badly cut off

<img width="333" alt="Screen Shot 2022-10-24 at 6 15 27 PM" src="https://user-images.githubusercontent.com/1479802/197640103-c9217c31-3b5e-4787-a44a-73477e20ecf9.png">

* We don't show the user's ENS Avatar if it's set
* The floating account center can obscure parts of the UI when scrolling
<img width="1288" alt="Screen Shot 2022-10-24 at 6 22 17 PM" src="https://user-images.githubusercontent.com/1479802/197640975-38e1e193-c40e-498a-a484-915728891918.png">

* The Loading Indicator shows up behind—or to the left— of the account center when switching networks:
![umbra-loading-indicator](https://user-images.githubusercontent.com/1479802/197640718-57a8e42a-d06d-46fd-bd11-1832f9363220.gif)

Reply:
Based on both of your feedback I don't think account center is mature enough for our use case. I can split out the work for #374 into a separate pr, and close this one.

For #315 I can add a custom solution that looks like [Uniswap](https://github.com/ScopeLift/umbra-protocol/issues/315#issuecomment-1225944841), but I think we can omit the change button along with the transaction history. Does that work?

Reply:
I agree with all of that. Though actually that "transaction history" section may be a good place to show the data in https://github.com/ScopeLift/umbra-protocol/issues/263, but we don't need to worry about that yet

Reply:
@alexkeating agree it sounds like Account Center might not work for the moment. Having not looked at the code for it at all, is there any world where maintaining a forked and modified version of account center (for now) is easier than rolling these solutions on our own? Maybe try to PR as many of our changes upstream as they'll accept, until it matures to the point we can switch back to the out of the box version? I don't have an opinion here, just asking for your assessment of that possibility vs. rolling our own.

Reply:
That is definitely an option, but I think rolling our own will be less effort because we already have many of the pieces to recreate the UI Matt suggested.